plugins/samson_ledger/test/lib/samson_ledger/client_test.rb

```diff
@@ -4,12 +4,23 @@
 SingleCov.covered!
 
 describe 'SamsonLedger::Client' do
-  let(:deploy) { Deploy.first }
+  ComparisonStruct  = Struct.new(:commits)
```

Comment:
do not define constants in tests, they will be global ... use let or local variables